lib/ClimaCorePlots/Project.toml

```
@@ -11,7 +11,7 @@ StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 TriplotBase = "981d1d27-644d-49a2-9326-4793e63143c3"
 
 [compat]
-ClimaCore = "0.11, 0.12, 0.13, 0.14"
+ClimaCore = "0.11, 0.12, 0.13, 0.14, 0.18"
```

```suggestion
ClimaCore = "0.11, 0.12, 0.13, 0.14"
```

We are not at version 0.18 yet :)

Does this need ClimaCore 0.14.18?

Same below for ClimaCoreTempestRemap

---

You're right, whoops! We can leave it as is since it's a patch update.

---

I was actually scratching my head about this when I was making the changes.